components/input-number/InputNumber.razor.cs

```diff
@@ -74,6 +74,9 @@ public TValue DefaultValue
         [Parameter]
         public EventCallback<TValue> OnChange { get; set; }
 
+        [Parameter]
+        public Action OnTextboxFocus { get; set; }
```

Comment:
Wouldn't it be better to use the same name as in `Input` component? I am talking about [`OnFocus`](https://github.com/ant-design-blazor/ant-design-blazor/blob/de2e6681dc594c92bbd4a3f793ee7b8e6bdbe8d0/components/input/Input.cs#L127)? And also `EventCallback<FocusEventArgs>` for the type.

Reply:
I would like to have all input-type components add the focus method. Can this be done together?

Reply:
@ElderJames Sure. Let me take a look at this later